### PR TITLE
POLICIA- SOLICITUD ASILO: Fixed Motivo o tipo de solicitud de la cita

### PR DESCRIPTION
**Pull request by Romster**
Hello. It turned out, that for "POLICIA - SOLICITUD ASILO" procedure there is an additional mandatory field
<textarea rows="4" size="35" maxlength="255" class="textarea_noborde error" title="Motivo o tipo de solicitud de la cita" required="required" id="txtObservaciones" name="txtObservaciones"></textarea>
I've added a workaround for it. Not the most elegant one, but it works.

### CODE REVIEW

File: bcncita/cita.py [hunk 1]
```diff
@@ -661,6 +660,14 @@ def phone_mail(driver: webdriver, context: CustomerProfile, retry: bool = False)
         element = driver.find_element_by_id("emailDOS")
         element.send_keys(context.email)
 
+        if context.operation_code == OperationType.SOLICITUD_ASILO:
+            # There is a mandatory field: "Motivo o tipo de solicitud de la cita"
+            try:
+                element = driver.find_element_by_id("txtObservaciones")
+                element.send_keys("solicitud de asilo")
```
[the remark this answers is not in the document]
Comment (Romster) [reply bcncita/cita.py]: refctored. wdyt?

File: bcncita/cita.py [hunk 2]
```diff
@@ -324,7 +324,6 @@ def solicitud_step2(driver: webdriver, context: CustomerProfile):
 
     return True
```
[the remark this answers is not in the document]
Comment (Romster) [reply bcncita/cita.py]: seems to be fixed

File: bcncita/cita.py [hunk 3]
```diff
@@ -175,6 +175,9 @@ class CustomerProfile:
     telegram_token: Optional[str] = None
     telegram_chat_id: Optional[str] = None
     wait_exact_time: Optional[list] = None  # [[minute, second]]
+    # "Motivo o tipo de solicitud de la cita" Required for some cases, like SOLICITUD_ASILO
+    # Example for SOLICITUD_ASILO: solicitud de asilo porque el documento caduca
+    reason_or_type: Optional[str] = None
```
[the remark this answers is not in the document]
Comment (Romster) [reply bcncita/cita.py:179]: added a link to the blog post

**Comment by Romster** [comment]
> Hey @Romster, thanks for the update on this! Could you please rebase the branch and add this stuff to readme?
> 
> https://github.com/cita-bot/cita-bot/blob/ed37ef61d8a952855ed9bb845b87d7eeecc3ac1c/bcncita/cita.py#L177-L179

Done. But it seems there are some issues with dependencies.

File: bcncita/cita.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (joanbr11) [reply bcncita/cita.py]: Hubo un cambio en los datos que piden para las citas. Por ende el cita bot no esta funcionando.

[the remark this answers is not in the document]
Comment (Miauremiau8) [reply bcncita/cita.py]: Contact me

**Comment by Miauremiau8** [comment]
No funciona, por favor me podría ayudaf

**Pull request merged**